src/components/button-panel.js

```diff
@@ -4,34 +4,34 @@ import Button from './button';
 
 const ButtonPanel = () => (
   <div>
-    <div className='row'>
-      <Button name='AC' className='col-md-3' />
-      <Button name='+/-' className='col-md-3' />
-      <Button name='%' className='col-md-3' />
-      <Button name='/' className='col-md-3' />
+    <div className='row btn-group'>
+      <Button name='AC' btn='btn' color='grey' />
+      <Button name='+/-' btn='btn' color='grey' />
+      <Button name='%' btn='btn' color='grey' />
+      <Button name='/' btn='btn' />
     </div>
-    <div className='row'>
-      <Button name='7' className='col-md-3' />
-      <Button name='8' className='col-md-3' />
-      <Button name='9' className='col-md-3' />
-      <Button name='X' className='col-md-3' />
+    <div className='row btn-group'>
+      <Button name='7' btn='btn' color='grey' />
+      <Button name='8' btn='btn' color='grey' />
+      <Button name='9' btn='btn' color='grey' />
+      <Button name='X' btn='btn' />
     </div>
-    <div className='row'>
-      <Button name='4' className='col-md-3' />
-      <Button name='5' className='col-md-3' />
-      <Button name='6' className='col-md-3' />
-      <Button name='-' className='col-md-3' />
+    <div className='row btn-group'>
+      <Button name='4' btn='btn' color='grey' />
+      <Button name='5' btn='btn' color='grey' />
+      <Button name='6' btn='btn' color='grey' />
+      <Button name='-' btn='btn' />
     </div>
-    <div className='row'>
-      <Button name='1' className='col-md-3' />
-      <Button name='2' className='col-md-3' />
-      <Button name='3' className='col-md-3' />
-      <Button name='+' className='col-md-3' />
+    <div className='row btn-group'>
+      <Button name='1' btn='btn' color='grey' />
+      <Button name='2' btn='btn' color='grey' />
+      <Button name='3' btn='btn' color='grey' />
+      <Button name='+' btn='btn' />
     </div>
-    <div className='row'>
-      <Button name='0' className='col-md-6' />
-      <Button name='.' className='col-md-3' />
-      <Button name='=' className='col-md-3' />
+    <div className='row btn-group'>
+      <Button name='0' btn='btn' wide color='grey' />
+      <Button name='.' btn='btn' color='grey' />
+      <Button name='=' btn='btn' />
     </div>
   </div>
 );
```

Comment:
- Let's try to make the rendering of your button panel more dynamic, you can render the entire component with the button components with a loop taking the info from an array or an object, that would save up several lines of code.

Reply:
**_I'll refactor and add the dynamic implementation of the button panel later. However, thanks for the suggestion @jofer86._**